src/AbstractFunctionReferenceFixer.php

```diff
@@ -67,7 +67,7 @@ protected function find($functionNameToSearch, Tokens $tokens, $start = 0, $end
         $matches = $tokens->findSequence($candidateSequence, $start, $end, false);
         if (null === $matches) {
             // not found, simply return without further attempts
-            return;
+            return null;
```

Comment:
There is no semantic difference, as defined by PHP's semantics. Both functions have return value `NULL`.